scripts/ci_utils.sh

```diff
@@ -146,6 +167,13 @@ function cpp_build_setup() {
         if (which apk) ; then
             # Packages already available in Docker build
             echo -n             # null op required for syntax
+        elif (which yum) ; then
+            sudo yum groupinstall -y "Development Tools"
```

Comment:
This seems to be useful for experimenting on an Amazon Linux instance. I.e. AFAIK we do not use RHEL-type OSes on the CI. If this is indeed used outside of the scope of the CI, we probably want to re-arrange the scripts to move this function in another file than this "ci_utils" script.

Reply:
moved to a general `build_utils.sh` script.